metadata-ingestion/src/datahub/ingestion/source/pulsar.py

```diff
@@ -89,7 +89,16 @@ def __init__(self, schema):
             logger.error(f"Invalid JSON schema: {schema_data}. Error: {str(e)}")
             avro_schema = {}
 
-        self.schema_name = avro_schema.get("namespace") + "." + avro_schema.get("name")
+        self.schema_name = "null"
+        if avro_schema.get("namespace") and avro_schema.get("name"):
+            self.schema_name = (
+                    avro_schema.get("namespace") + "." + avro_schema.get("name")
```

Comment:
The latest build failed because of a format issue, which should be fixed with:

```suggestion
                avro_schema.get("namespace") + "." + avro_schema.get("name")
```